doc/designs/ddd/README.md

```diff
@@ -86,8 +86,8 @@ the applications:
 | pgbouncer        | A |      AOSF, BIOc  |
 
 * Blk: Whether the application uses blocking or non-blocking I/O.
-  * S: Blocking
-  * A: Nonblocking
+  * S: Blocking (Synchronous)
+  * A: Nonblocking (Asynchronous)
```

Comment:
This is not a synonym and I think it might be confusing to some.

Reply:
okay.  Does anyone have a suggestion on what to do here?

Reply:
I don't really mind this terminology.